src/Components/SystemDetail/SystemRecommendations.scss

```diff
@@ -0,0 +1,3 @@
+.recsTitle {
```

Comment:
No need to do it like this you can change the `Stack` component to have gutter `<Stack hasGutter>`

Reply:
Done, thank you @karelhala!